docs/reference/remap.cue

```diff
@@ -51,7 +51,9 @@ remap: {
 			arguments: [...#Argument] // Allow for empty list
 			internal_failure_reasons: [...string]
 			return: [#RemapReturnTypes, ...#RemapReturnTypes]
-			category:    "Array" | "Check" | "Coerce" | "Decode" | "Encode" | "Enumerate" | "Event" | "Hash" | "IP" | "Log" | "Map" | "Number" | "Parse" | "Random" | "String" | "Test" | "Timestamp"
+			category: "Array" | "Check" | "Coerce" | "Decode" | "Encode" | "Enumerate" | "Event" |
```

Comment:
Just noting for future readers that the list is the same here :smile:

Reply:
Haha, yes. No time like the present for some housekeeping.